dev/cognitiveservices/data-plane/Language/analyzetext.json

```diff
@@ -2680,8 +3562,36 @@
         "piiCategories": {
           "$ref": "#/definitions/PiiCategories"
         },
+        "excludePiiCategories": {
+          "$ref": "#/definitions/PiiCategories"
+        },
         "stringIndexType": {
           "$ref": "common.json#/definitions/StringIndexType"
+        },
+        "redactionCharacter": {
```

Comment:
Should this be pulled into its own definition in common.json to share with conversation?

Reply:
I'll try to address this and also modify the exclusion list (since we don't want default and all) later today/early tomorrow.